Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -116,6 +119,8 @@ bool PlatformerObjectBehavior::UpdateProperty(
       behaviorContent.SetAttribute("gravity", value.To<double>());
     else if (name == _("Max. falling speed"))
       behaviorContent.SetAttribute("maxFallingSpeed", value.To<double>());
+    else if (name == _("Ladder Climbing Speed"))
```

Comment:
'S' should be s here because other properties have also been defined the same way!

Reply:
This solves the issue. Sorry, I couldn't notice it; I don't really know why :/